yaksh/views.py

```diff
@@ -306,9 +308,9 @@ def prof_manage(request, msg=None):
         for paper in question_papers:
             answer_papers = AnswerPaper.objects.filter(question_paper=paper)
             users_passed = AnswerPaper.objects.filter(question_paper=paper,
-                    passed=True).count()
+                    passed=True).distinct().count()
```

Comment:
What ```distinct()``` will do here?
I think it can do nothing here, because our attempt number is always different.